reader.py

```diff
@@ -0,0 +1,38 @@
+import http.client
+import json
+import datetime
+from util import SortUtil
+
+#Reads the data from the resources
+class Reader:
+    global start 
+    start = datetime.datetime.now()
```

Comment:
You can put  start = datetime.datetime.now() and diff = datetime.datetime.now() - start outside of this class.

Reply:
start = datetime.datetime.now() and diff = datetime.datetime.now() - start is now in main